animalfacts.py

```diff
@@ -2739,6 +2740,16 @@ def animalfactsbot(reddit):
     'Tardigrades work as pioneer species by inhabiting new developing environments in which to live. Their presence attracts other invertebrates and predators to populate the space.',
     'Tardigrades are the first known animal to survive in space.'
     )
+TARSIER_FACTS = (
+    'A Tarsier is one of the known smallest primates. It is an ancient ancestor of a modern day monkey.',
+    'A Tarsier does not grow bigger than a man''s hand. It is about 15cm in height and between 115 and 130g in weight.',
+    'A Tarsier have gray fur and a nearly naked tail.',
+    'A Tarsier is recognizable due to its huge eyes and its long feet. It has the largest eyes, proportionate to its body size, of any animal on the planet.',
+    'Their large eyes enables them to have acute night vision that makes them a good hunter.',
+    'Their long feet enables them to jump from tree to tree up to 3 meters in distance.',
+    'A Tarsier is mostly active at night and lives on a diet of insects.',
+    'A Tarsier can usually be found in the beautiful archipelago called the Philippines.'
```

Comment:
Could you pop a comma on the end of this line to match the other tuples? Otherwise these facts are all great!

Reply:
Thanks for your edits. I'm totally fine with it.